src/main/java/org/dependencytrack/event/kafka/KafkaTopic.java

```diff
@@ -33,7 +36,12 @@ public enum KafkaTopic {
     }
 
     public String getName() {
-        return name;
+        var prefix = "";
+        SmallRyeConfig config = ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
+        var prefixConfig = config.getConfigValue("api.topic.prefix");
+        if (prefixConfig.getValue() != null)
+            prefix = prefixConfig.getValue();
+        return prefix + name;
```

Comment:
There already is a mechanism to read configuration, no need to introduce SmallRye Config as additional dependency. See [`ConfigKey`](https://github.com/DependencyTrack/hyades-apiserver/blob/main/src/main/java/org/dependencytrack/common/ConfigKey.java), which can be accessed with `Config.getInstance().getProperty(ConfigKey.FOOBAR)`.